common/models/access-token.js

```diff
@@ -168,9 +168,12 @@ module.exports = function(AccessToken) {
     var length;
     var id;
 
-    params = params.concat(['access_token']);
-    headers = headers.concat(['X-Access-Token', 'authorization']);
-    cookies = cookies.concat(['access_token', 'authorization']);
+    // https://github.com/strongloop/loopback/issues/1326
+    if (options.searchDefaultTokenKeys === undefined || options.searchDefaultTokenKeys === true) {
```

Comment:
We usually use a simpler check `options.searchDefaultTokenKeys !== false`. It works well for `true`, `false` and `undefined`, even though it behaves differently for edge cases like `null` or non-boolean value.